Assets/Scripts/PlayerCharacter/PlayerController.cs

```diff
@@ -35,6 +36,8 @@ public enum RotationMode
         private Rigidbody rb;
 
         private DefaultInputActions actions;
+        private static readonly int Speed = Animator.StringToHash("speed");
+        private static readonly int Jump = Animator.StringToHash("jump");
```

Comment:
prefix variables with `Prop` or `Animator` or something so it's obvious that this is Animator Property